src/data_model/eng2math.jl

```diff
@@ -731,25 +620,28 @@ function _map_eng2math_voltage_source!(data_math::Dict{String,<:Any}, data_eng::
         math_obj["pmax"] = get(eng_obj, "pg_ub", fill( Inf, nconductors))
         math_obj["qmin"] = get(eng_obj, "qg_lb", fill(-Inf, nconductors))
         math_obj["qmax"] = get(eng_obj, "qg_ub", fill( Inf, nconductors))
-        math_obj["configuration"] = WYE
+        math_obj["connections"] = eng_obj["connections"]
+        math_obj["configuration"] = get(eng_obj, "configuration", WYE)
         math_obj["source_id"] = "_virtual_gen.$(eng_obj["source_id"])"
 
         _add_gen_cost_model!(math_obj, eng_obj)
 
         map_to = "gen.$(math_obj["index"])"
 
         if !all(isapprox.(get(eng_obj, "rs", zeros(1, 1)), 0)) && !all(isapprox.(get(eng_obj, "xs", zeros(1, 1)), 0))
+            f_bus = deepcopy(data_math["bus"]["$(math_obj["gen_bus"])"])
+
             bus_obj = Dict{String,Any}(
                 "bus_i" => length(data_math["bus"])+1,
                 "index" => length(data_math["bus"])+1,
-                "terminals" => collect(1:nconductors+1),
-                "grounded" => [fill(false, nconductors)..., true],
+                "terminals" => f_bus["terminals"],
+                "grounded" => f_bus["grounded"],
```

Comment:
As we discussed in the previous PR, this is inconsistent with my mental model of what a voltage source represents. We can come back to this in v0.10 to decide what this should be, after discussing what it actually represents. In the meanwhile, if you decide to keep these changes, be aware this will break `PMD4W.jl` and you should use an older version of `PMD` with it for now.

Reply:
I have made some corrections to this section, and have confirmed that PMD4W.jl produces the same result with both PMD master and this branch on the Iowa case, the only change I had to make was to add the kwarg `project_phases=true` to `transform_data_model`